steps/property-inspection/antisocial-behaviour.tsx

```diff
@@ -78,7 +78,7 @@ const step = {
             name: "tenant-understands",
             legend: (
               <FieldsetLegend>
-                Does the tenant understand about antisocial behaviour?
+                Have you discussed antisocial behaviour with tenant?
```

Comment:
```suggestion
                Have you discussed antisocial behaviour with the tenant?
```

Reply:
Jira note : change question text to: Have you discussed antisocial behaviour with tenant?
Still include the 'the?

Reply:
Yup.